examples/cpu_load.py

```diff
@@ -9,13 +9,13 @@
 except ImportError:
     exit("This script requires the psutil module\nInstall with: sudo pip install psutil")
 
-from blinkt import set_clear_on_exit, set_brightness, set_pixel, show
+from blinkt import set_clear_on_exit, set_brightness, set_pixel, show, NUM_PIXELS
```

Comment:
It's only tangentially related to your PR, but I think to better support this change, all instances of explicit imports should be replaced with:

```
import blinkt
```

And then `blinkt.set_clear_on_exit`, `blinkt.set_brightness`, `blinkt.set_pixel`, `blinkt.NUM_PIXELS`, etc.